test_regress/t/t_var_xref_gen.pl

```diff
@@ -14,5 +14,8 @@
      verilator_flags2 => ["--debug-check"],
     );
 
-ok(1);
+if (-e "$Self->{obj_dir}/Vt_var_xref_gen.xml") {
+    ok(1);
+}
```

Comment:
Think it would be better ot make a new test_regress/t/t_var_xref_gen.out file and compare it (see the files_identical syntax in the other xml tests).  This might also cover some of the missing items I pointed out earlier.